### ISSUE TITLE
DATABASE: Update fiadb_fvs_variant table in biosum_ref.accdb

### ISSUE THREAD

**Issue by lbross** [USFS-PNW/Fia-Biosum-Manager#317]
From @jsfried: I did a quick check in Oregon and found that the VARIANT code in PLOTGEOM is the same as the one in our BioSum look-up table for all but 2 of 31505 plot locations. However, the LOC_CODE is different (701 vs 702) for 1688 of those cases. Not sure how often PLOTGEOM is updated but out BioSum internal fiadb_fvs_variant table hasn’t been updated in probably 5-6 years (I never thought to). I now see that it does need updating because FVS tweaks its maps a bit over time, apparently, and because new plots get added when spatial intensifications are undertaken (as they were in Oregon over the last few years).

Optional: Get the default move distance (fiadb_fvs_variant.MoveDistance_ft) as the midpoint of the road distance class in PLOT.RDDISTCD given that this distance is populated from the actual plot location, not from a point that can be up to .71 miles away.

We would like to include this updated table in v5.11.0. Another issue will be opened for a longer term fix to populate the VARIANT directly from an FIADB table when a plot is loaded.

**Comment by lbross** [comment]
The versioning in the biosum_ref.accdb needs to be updated when this is ready for deployment. This tells BioSum to pull down the updated reference database. This database is shared between projects so this only happens once. Any pre-existing customizations to tables in this database will be overwritten/lost.

**Comment by lbross** [comment]
From Nathan: Here’s a screenshot of the REF_VERSION table in biosum_ref.accdb in my ~\AppData\Roaming\FIABiosum folder, which confirms that the version number and date are correct. I loaded a bunch of plots for the Mount Hood WCS landscape and had no problem having the FVS variant (EC or WC) being assigned.  Sounds like this item (Issue #317) can be crossed of, correct?

**Comment by lbross** [comment]
@jsfried has additional updates for this table. He would like to leave all plots (including only nonforest conditions and nonsampled conditions) in the table. He posted an updated table to BioSumBox\Development\BioSumRef\fiadb_fvs_variant\FFV24a.accdb. At the August 15 DEV call we agreed to hold of on deploying this new table until v5.11.1.

**Comment by jsfried** [comment]
BioSum now will allow you to pull in (as far as master.mdb) non-forest conditions INCLUDING those from plots that are entirely non-forest. This is possible because non-forest plots in CA, OR and WA, on NFS lands, which DO get measured by FIA, are not included in the FIADB_FVS_VARIANT table so can be assigned a variant at load time.   BUT it won't add them to your FVSIn file (because there is not  a forest type or site index to give to FVS or even a habtypcd because those are not collected on nonforest plots). So it will be up to the (ambitious and dextrous) user to populate missing data and swap the status to forested (before making the FVSIn) if they want to  model afforestation or even be able to model tree growth on these plots (that have trees in master.mdb but not in the FVSIn that will be created by BioSum unless furhter data massage is attempted -- e.g., changing cond_status_cd to 1). It may be the case that simply changing the cond_status_cd will be enough to get it into the FVSIn-- the FVS_STANDINIT_COND and FVS_TREEINIT_COND tables in the FIADB have these conditions and trees populated (but with no site productivity info).

**Comment by lbross** [comment]
@jsfried I'm moderately confused by the above comment. This seems to be a documentation issue more than a system issue. You are seeing the updated ref_master.accdb, correct?
Also I think that not should be replaced with now in this sentence. "This is possible because non-forest plots in CA, OR and WA, on NFS lands, which DO get measured by FIA, are **now** included in the FIADB_FVS_VARIANT table so can be assigned a variant at load time."

**Comment by jsfried** [comment]
Yes-- the ref_master is correctly updated. I wanted to understand the behavior of those nonforest plots as they propagate through the BioSum workflow in case someone chooses to load them. So I experimented and provide some docs above (which need to go in the users guide, @Cat-Ducat-USDA, to let folks know that just loading non-forest plots into BioSum does not guarantee that they will make it all the way into FVS or be modeled as desired in FVS without futher tweaking (which users are free to pursue, and we are free not to support). The challenge with the FIADB_FVS_VARIANT table is that any plot (and any condition on any plot) can switch from forested to not, and vice versa, at any visit. In the interest of ensuring automation of variant assignment and yarding distance info, we wanted a more expanded table, at least covering NFS lands where we know that FIA data is collected (inclduing trees, sometimes) despite the non-forest categorization of the land status. I corrected the typo above- thanks for pointing it out.
